db/migrate/20230727072004_create_keywords.rb

```diff
@@ -0,0 +1,26 @@
+class CreateKeywords < ActiveRecord::Migration[7.0]
+  def change
+    create_enum  :keyword_status, %w[processing succeeded failed]
+
+    create_table :keywords, id: :serial do |t|
+      t.string :content, limit: 255, null: false
+      t.enum :status, enum_type: 'keyword_status', default: 'processing', null: false
```

Comment:
should default as `pending` instead of `processing` 🤔 it will change to processing when we actual start the process 🤔

Reply:
Good point. Fixed in 5e498403fb9cb5f3583c9e5f701826ce34f60c42